js/wee.history.js

```diff
@@ -367,6 +389,15 @@
 			request.url = url;
 			conf.request = request;
 
+			if (conf.begin && W.$exec(conf.begin, {
```

Comment:
Okay, that sounds good. I figured as much, but I wanted to make sure I wasn't overlooking something.